spec/views/posts/show_spec.rb

```diff
@@ -0,0 +1,38 @@
+require 'rails_helper'
+
+RSpec.describe 'User post', type: :feature do
+  before(:example) do
+    @user = User.create(name: 'Elon Musk', photo: 'link/goes/here', bio: 'this is my bio')
+    @post_one = Post.create(title: 'My first post', text: 'Body text of the post', author_id: @user.id)
+    Post.create(title: 'My second post', text: 'Body text of the post', author_id: @user.id)
+    Post.create(title: 'My third post', text: 'Body text of the post', author_id: @user.id)
+    Post.create(title: 'My fourth post', text: 'Body text of the post', author_id: @user.id)
+    Comment.create(text: 'My first awesome comment straight from Kenya', author_id: @user.id, post_id: @post_one.id)
+
+    visit user_post_path(@user, @post_one.id)
+  end
+
+  it 'renders who wrote the post ' do
+    expect(page.html).to include(@user.name)
+  end
```

Comment:
- [ ] As per this [requirement](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/intergration_spec_and_n_1_project.md#:~:text=I%20can%20see%20the%20post%27s%20title.), Please add a Test that asserts you can see the post's title.

Reply:
Hi @Omar-Muhamad, thank you for taking the time to leave this excellent review. The changes above have been applied. Cheers